s3.tf

```
@@ -1,3 +1,3 @@
-resource "aws_s3_bucket" "dashboard_reports_bucket" {
-  bucket = local.reports_bucket_name
+resource "aws_s3_bucket" "application_bucket" {
```

```suggestion
resource "aws_s3_bucket" "application" {
```
nitpick -- We shouldn't repeat resource type in resource name